tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -244,6 +366,8 @@ teardown() {
 
     [ "${SNAPSHOTTER:-}" = "nydus" ] || skip "None snapshotter was found but this test requires one"
 
-    kubectl describe pod "$pod_name"
+    # TODO - where is pod_name supposed to be set?
```

Comment:
Was curious and digging... Maybe this is where/how the pod_name is gotten? k8s_create_pod has a local reference to it briefly https://github.com/kata-containers/kata-containers/blob/893fd2b59cc31518f8a127c9611e3e8265d9bdfd/tests/integration/kubernetes/lib.sh#L51

Reply:
Yeah, I followed that path too, but the `local` suggests that it wouldn't be available. It might have been that it never worked and pulling in `load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"` has pulled in `set -o nounset` somewhere where it wasn't before and that's why we are only seeing the failure now.

Reply:
Oh, I see... Frankly leaving the TODO seems smart at this point, then.